### DISCUSSION TITLE
Help! Need some brainstorming ideas if possible?

### DISCUSSION THREAD

**Discussion by lrjsoie** [community/community#40964] [New to GitHub]
Long Long time ago I was signed up for an AMP project. The person stated they would explain what it was. I did not agree unless I was aware of what the project was, and the intent behind the project. Many years have passed and since he will not acknowledge signing me up through an AWS....I was wondering how would I be able to be removed from a project?

**Comment by ossanna16** [comment]
Hi there @lrjsoie and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
